src/components/quiz/Answer.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import Button from "../Button";
+import "./Answer.scss";
+
+interface AnswerProps {
+  isCorrect: boolean;
+  answerText: string;
+  handleAnswerOptionClick(isCorrect: boolean): void;
```

Comment:
I think this could be renamed to have a more clear name, something even as simple as `onClick`!